### PR TITLE
Feature/overriding simplicity

### PR DESCRIPTION
**Pull request by andrii-onufriichuk**

### CODE REVIEW

File: view/frontend/web/template/payment/card-form.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--Pay By Card Form-->
+<!-- ko if: (getCode() == 'rvvup_CARD' && rvvup_parameters.settings.card.flow === 'INLINE') -->
+<form id="st-form" method="POST" class="rvvup-st-form">
```
Comment (aashwin-rvvup) [inline view/frontend/web/template/payment/card-form.html:3]:
```suggestion
<form id="st-form" method="POST" class="rvvup-card-form">
```

File: view/frontend/web/template/payment/iframe-modal.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div style="display: none;" data-bind="attr: {'id': getModalId()}" class="rvvup-iframe-modal">
```
Comment (aashwin-rvvup) [inline view/frontend/web/template/payment/iframe-modal.html:1]:
```suggestion
<div style="display: none;" data-bind="attr: {'id': getModalId()}" class="rvvup-modal-container">
```

File: view/frontend/web/template/payment/payment-title.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="payment-method-title field choice rvvup-payment-method-title">
```
Comment (aashwin-rvvup) [inline view/frontend/web/template/payment/payment-title.html:1]: rvvup-payment-method-container

File: view/frontend/web/template/payment/payment-title.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    <input type="radio"
+           name="payment[method]"
+           class="radio"
+           data-bind="attr: {'id': getCode()}, value: getCode(), checked: isChecked, click: selectPaymentMethod, visible: isRadioButtonVisible()">
+    <label data-bind="attr: {'for': getCode()}" class="label">
+        <span data-bind="text: getTitle()"></span>
+    </label>
+    <img class="rvvup-logo" data-bind="attr: {'src': getLogoUrl(),'class': getCode().replace('_','-').toLowerCase() + '-img'}" />
+    <br>
+    <span class="rvvup-description" data-bind="text: getDescription()"></span>
```
Comment (aashwin-rvvup) [inline view/frontend/web/template/payment/payment-title.html:11]:
```suggestion
    <span class="rvvup-payment-method-description" data-bind="text: getDescription()"></span>
```

File: view/frontend/web/template/payment/paypal-button.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+                 data-bind="attr: {'id': getPayPalId()}, css: {disabled: !isPlaceOrderActionAllowed()}">
+            </div>
+        </div>
+    </div>
+</div>
+<!--/ko-->
+
+<!-- ko if: (getUsePlaceOrderStyling() == false) -->
+<!-- PayPal Place Order button, show if PayPal option is selected -->
+<div data-bind="attr: {class: getPaypalBlockStyling()}, style: {display: shouldDisplayPayPalButton() ? 'block' : 'none'
```
Comment (aashwin-rvvup) [inline view/frontend/web/template/payment/paypal-button.html:16]: I think this data-bind will overwrite the class provided?

Comment (andrii-onufriichuk) [reply view/frontend/web/template/payment/paypal-button.html:16]: Ah yes, it's set through admin. Removed it.

**Review by aashwin-rvvup — APPROVED** [review]

**Pull request merged**